resources/public/css/site.css

```diff
@@ -5,12 +5,13 @@ html {
 
 body {
   font-family: Voces, "Helvetica Neue", Helvetica, Arial, sans-serif;
-  padding-top: 12px;
-  padding-bottom: 70px;
+// padding-top: 12px;
```

Comment:
Single-line comments are unofficial CSS syntax. Moreover, comments should not be committed.